From 637475ed62eba7486f2a3781d22c2bf0bd1179ba Mon Sep 17 00:00:00 2001
From: Bhawanpreet Lakha <Bhawanpreet.Lakha@amd.com>
Date: Wed, 7 Aug 2019 11:23:30 -0400
Subject: [PATCH 3870/4256] drm/amd/display: Add hdcp to Kconfig

[Why]
HDCP is not fully finished, so we need to be able to
build and run the driver without it.

[How]
Add a Kconfig to toggle it

Signed-off-by: Bhawanpreet Lakha <Bhawanpreet.Lakha@amd.com>
Reviewed-by: Harry Wentland <harry.wentland@amd.com>
Signed-off-by: Alex Deucher <alexander.deucher@amd.com>
---
 drivers/gpu/drm/amd/display/Kconfig | 8 ++++++++
 1 file changed, 8 insertions(+)

diff --git a/drivers/gpu/drm/amd/display/Kconfig b/drivers/gpu/drm/amd/display/Kconfig
index 8154fd637afb..b4504257873a 100644
--- a/drivers/gpu/drm/amd/display/Kconfig
+++ b/drivers/gpu/drm/amd/display/Kconfig
@@ -43,6 +43,14 @@ config DRM_AMD_DC_DSC_SUPPORT
 	    Choose this option if you want to have
 	    Dynamic Stream Compression support
 
+config DRM_AMD_DC_HDCP
+        bool "Enable HDCP support in DC"
+        depends on DRM_AMD_DC
+        help
+         Choose this option
+         if you want to support
+         HDCP authentication
+
 config DEBUG_KERNEL_DC
 	bool "Enable kgdb break in DC"
 	depends on DRM_AMD_DC
-- 
2.17.1

